src/main/steps/tasklistresponse/summary/postControllerAfterPcq.ts

```diff
@@ -0,0 +1,52 @@
+import { Logger } from '@hmcts/nodejs-logging';
+import { Response } from 'express';
+
+import { CosApiClient } from '../../../app/case/CosApiClient';
+import { CaseEvent, CaseType, PartyType } from '../../../app/case/definition';
+import { AppRequest } from '../../../app/controller/AppRequest';
+import { AnyObject, PostController } from '../../../app/controller/PostController';
+import { FormFields, FormFieldsFn } from '../../../app/form/Form';
+import { CA_RESPONDENT_RESPONSE_CONFIRMATION } from '../../urls';
+import { getPartyDetails, mapDataInSession } from '../utils';
+
+const logger = Logger.getLogger('ResponseSummaryConfirmationPostController');
+
+export class ResponseSummaryConfirmationPostController extends PostController<AnyObject> {
+  constructor(protected readonly fields: FormFields | FormFieldsFn) {
+    super(fields);
+  }
+
+  public async post(req: AppRequest, res: Response): Promise<void> {
```

Comment:
As said already have all the logic in one method that will redirect the user to PCQ site based on request type / context. Get rid of this file